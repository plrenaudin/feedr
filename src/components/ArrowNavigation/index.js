import Icon from "../Icon";

const ArrowNavigation = ({ onLeft, onRight }) => (
  <section class="hide-on-mobile">
    <div class="left-right">
      <div class="left" onClick={onLeft}>
        <Icon name="arrow" />
      </div>
      <div class="right" onClick={onRight}>
        <Icon name="arrow" />
      </div>
    </div>
  </section>
);

export default ArrowNavigation;
