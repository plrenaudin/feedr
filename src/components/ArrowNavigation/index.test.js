import { render } from "inferno";
import ArrowNavigation from ".";

it("Matches the snapshot", () => {
  const div = document.createElement("div");

  expect(
    render(<ArrowNavigation onLeft={() => {}} onRight={() => {}} />, div).dom
  ).toMatchSnapshot();
});
