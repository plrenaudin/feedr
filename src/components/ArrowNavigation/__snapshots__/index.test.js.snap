// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot 1`] = `
<section
  class="hide-on-mobile"
>
  <div
    class="left-right"
  >
    <div
      class="left"
    >
      <span
        class="icon-wrapper"
      >
        <svg
          class="icon"
        >
          <use
            xlink:href="#icon-arrow"
          />
        </svg>
      </span>
    </div>
    <div
      class="right"
    >
      <span
        class="icon-wrapper"
      >
        <svg
          class="icon"
        >
          <use
            xlink:href="#icon-arrow"
          />
        </svg>
      </span>
    </div>
  </div>
</section>
`;
